### PR TITLE
Doc updates for runner.sh and resource pool selection.

### PR DESCRIPTION
**Pull request by hulquest**
Added some updates to the readme about downloading runner.zip from the projects package location.  Also tried to clarify how clusters could be specified as a resource pool.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,18 +22,28 @@ There are 2 ways to run EZ-Rancher:
...
+#### Data Center
+
+The `vm_datacenter` should be set to the vSphere DataCenter that has access to the remaining vSphere parameters.
+
+#### Resource Pool
+
+The `vsphere_resource_pool` can be the name of a vSphere ResourcePool or Cluster.  If using a Cluster, then be sure to add "/Resources" after the name of the Cluster, like `gpu_cluster/Resources`.  The default value for this parameter is `Resources` which works fine for data centers that have a single cluster.
 
-The `vm_datastore` must be set to a VMFS datastore to support disk resizing.
+#### Data Store
```
Comment (bgentry-ntap) [inline README.md:44]: nit: datastore is one word

File: README.md [hunk 2]
```diff
@@ -45,7 +55,7 @@ The `rancher_server_url` input must resolve to one or more of the worker node IP
 
 If DHCP is used (default), this can be done after the deployment completes and the worker nodes receive an IP from the `vm_network`.
 
-#### Auto DNS URL
+#### Auto DNS URL (DHCP)
```
Comment (bgentry-ntap) [inline README.md:58]: I think this would also apply to static IPs

File: README.md [hunk 3]
```diff
@@ -190,21 +209,21 @@ cluster_nodes = [
...
 
-## Accessing the Rancher UI
+## Accessing Rancher UI
 
 In order to access the Rancher UI via your chosen `rancher_server_url`, you must ensure that a valid DNS record exists, that resolves to one or more of your cluster node IPs.
 
 Once the DNS record is in place, the Rancher UI will become accessible at the location specified for `rancher_server_url`.
 
-### Automatic DNS Names
+### Automatic DNS Names (DHCP)
```
Comment (bgentry-ntap) [inline README.md:218]: same comment on dhcp

File: README.md [hunk 4]
```diff
@@ -94,9 +102,20 @@ terraform apply -var-file=rancher.tfvars terraform/vsphere-rancher
...
 
+### Runner Script
+
+Download the `runner.zip` package from our [Releases](https://github.com/NetApp/ez-rancher/releases) page to run the installation from a shell script;  Docker remains a prerequisite.
+
+* Select the release (the most recent is recommended) and `wget https://github.com/NetApp/ez-rancher/releases/_release_/runner.zip`
+* Unzip the artifact.
+* Copy `rancher.tfvars.example` to `rancher.tfvars`
+* Tune `rancher.tfvars` for your environment.
+* Run `./runner.sh` to install Rancher Server
```
Comment (bgentry-ntap) [inline README.md:113]: you need to specify the `apply` operation or we need to default that in the runner script

Comment (hulquest) [reply README.md:113]: Ok, I thought apply was the default...but didn't check the script.  I'll update here.

**Pull request merged**